acrclient/client.py

```diff
@@ -155,3 +156,14 @@ def get_bm_cs_projects_results(
             params=params,
             **kwargs,
         ).get("data")
+
+    def get_external_metadata_tracks(
+        self, params: Optional[GetExternalMetadataTracksParams] = None
+    ) -> list:
+        try:
+            return self.json(path="/api/external-metadata/tracks", params=params).get(
+                "data"
+            )
+        except HTTPError as error:
+            print(error)
```

Comment:
imo libs should not have side effects like `print()`
```suggestion
```

Reply:
Yes, I just pushed the code to have it somewhere. I agree :)